drivers/wdm/audio/backpln/portcls/version.cpp

```diff
@@ -6,13 +6,9 @@
  * PROGRAMMER:      Johannes Anderwald
  */
 
-#include "private.hpp"
 
-#ifndef YDEBUG
-#define NDEBUG
-#endif
+#include "private.hpp"
```

Comment:
strip one newline above #include "private.hpp" as it will help git compare to better align what should be aligned.

Reply:
Fixed.